packages/KSelect/KSelect.vue

```diff
@@ -322,7 +365,8 @@ export default {
       return false
     },
     filteredItems: function () {
-      return this.filterFunc({ items: this.selectItems, query: this.filterStr })
+      // For autosuggest, items don't need to be filtered internally
```

Comment:
So the `filterFunc` prop does not work with `autosuggest` enabled, correct?

If this is the case, we need to note this under the documentation for `filterFunc` in `select.md`

Reply:
Right. I added a note.